spec/venue_spec.rb

```diff
@@ -44,12 +43,51 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
+      
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  describe '#over_capacity?' do
+    it 'returns false appropriately' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+
+      expect(venue.over_capacity?).to eq(false)
+    end
+
+    it 'returns true appropriately' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('James')
+      venue.add_patron('Cat')
+
+      expect(venue.over_capacity?).to eq(true)
+    end
```

Comment:
Good job with these tests! You thought through both the true and false paths. Something else to consider is what this method should do when the number patrons is exactly equal to the capacity...